src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -21,6 +22,14 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public class WwwAuthenticateParameters
     {
+        private static readonly ISet<string> s_knownAuthenticationSchemes = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
+
+        static WwwAuthenticateParameters()
```

Comment:
avoid using static ctor, they are difficult to reason with. You can initialize the HashSet at line 25.